### PR TITLE
4.6.1 Release notes - copy edit

### PR DESCRIPTION
**Pull request by elstoc**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -31,75 +31,75 @@ Since darktable 4.4:
...
 
-- n/a
+- Documentation is now complete for darktable 4.6 including translation
+  into Ukrainian and Dutch.
 
 ## Performance Improvements
 
-- Fix some OpenCL code paths that could produce garbled or unprocessed
+- Fixed some OpenCL code paths that could produce garbled or unprocessed
   output without back reporting the the pixel-pipe.
```
Comment (elstoc) [inline RELEASE_NOTES.md:42]: BTW I don't really understand what "without back reporting the the pixel-pipe" is trying to say.

Comment (TurboGit) [reply RELEASE_NOTES.md:42]: I did understood it as "without reporting back to the pixel-pipe". Meaning the pixel-pipe is not aware of the OpenCL failure and could not take a proper decision like fallback on CPU path.

Comment (elstoc) [reply RELEASE_NOTES.md:42]: Ok that makes more sense. I'll change it

**Review by TurboGit — APPROVED** [review]
Thanks!

**Pull request merged**